lib/fog/linode/models/compute/kernels.rb

```diff
@@ -11,20 +11,27 @@ def all
           load kernels
         end
 
+        def all_kvm
+          load kernels(:isKVM => 1)
+        end
+
+        def all_xen
+          load kernels(:isXen => 1)
+        end
```

Comment:
Not sure about those methods. Maybe we should have filters on the `all` method?

```ruby
def all(options = {})
  load kernels(options)
end
```

What do you think?